tests/test_legistar_scraper.py

```diff
@@ -0,0 +1,164 @@
+import pytest
```

Comment:
if you rename this file to something that doesn't start with `test` you won't have to explicitly exclude it. `legistar_integration.py`

would be a fine name.